ci-operator/step-registry/openshift/e2e/azure/csi/secrets-store/vault/openshift-e2e-azure-csi-secrets-store-vault-workflow.yaml

```diff
@@ -0,0 +1,27 @@
+workflow:
+  as: openshift-e2e-azure-csi-secrets-store-vault
+  steps:
+    pre:
+    - chain: ipi-azure-pre
+    - ref: optional-operators-subscribe
+    - ref: storage-create-csi-secrets-store
+    - ref: storage-conf-wait-for-csi-driver
+    - ref: storage-obj-save
+    test:
+    - as: test-e2e-vault
+      cli: latest
+      commands: make e2e-vault
+      from: src
```

Comment:
> make: *** No rule to make target 'e2e-vault'.  Stop.

`from: src` refers to the operator src here, but this workflow needs the driver src to run [e2e-vault](https://github.com/openshift/secrets-store-csi-driver/blob/556773e6d44bc303fff28418f5e06760bcb80d47/Makefile#L481-L483).

You need to define a src image to be built by the driver, and then use that image here. I would try adding something like this to `ci-operator/config/openshift/secrets-store-csi-driver/openshift-secrets-store-csi-driver-main.yaml` in the `images` section:

```
- dockerfile_literal: |-
    FROM src
  from: src
  to: secrets-store-csi-driver-test
```

And then once we have that source image for the driver, your workflow can reference it with:

```
from: secrets-store-csi-driver-test
```

At least I think that will work... might take a little trial-and-error. (see also: [build-inputs docs](https://docs.ci.openshift.org/docs/architecture/ci-operator/#build-inputs))